providers/default.rb

```diff
@@ -58,6 +58,7 @@
     mode '0755'
     backup false
 
+    notifies :stop, "service[tomcat_#{new_resource.instance_name}]", :immediately
```

Comment:
Seems to work fine, we see this on a fresh VM where the init script hasn't yet been created:

> * service[tomcat_service-8080] action stop (up to date)